avro-fastserde/src/test/java/com/linkedin/avro/fastserde/micro/benchmark/AvroGenericDeserializer.java

```diff
@@ -0,0 +1,68 @@
+package com.linkedin.avro.fastserde.micro.benchmark;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericDatumReader;
+import org.apache.avro.io.BinaryDecoder;
+import org.apache.avro.io.DatumReader;
+import org.apache.avro.io.DecoderFactory;
+
+
+public class AvroGenericDeserializer<V> {
+
+  /**
+   * This default is intended for the backend. The client overrides this, and its default is defined in
```

Comment:
This comment is not relevant in this context.

Reply:
Removed.